clients/include/testing_matmul.hpp

```diff
@@ -832,8 +832,9 @@ void testing_matmul(const Arguments& arg)
 
     // Get Heuristic results
     std::vector<hipblasLtMatmulHeuristicResult_t> heuristicResult{1};
```

Comment:
cannot hard-code it as 1 element now

Reply:
>> besides that, don't use vector for non-ext path.
This should open another PR for that instead of here cause it's not related to the commit.